src/modules/Logs/api.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-const API_PREFIX = 'https://info1.edgesecure.co:8444/v1/'
+const API_PREFIX = 'https://logs1.edge.app:8444/v1/'
```

Comment:
not part of this pull request

as well remove the port for logs1

Reply:
My branches are polluting each other. Removed.